README.md

```diff
@@ -1,36 +1,61 @@
-# Coordinator
+# XAIN Federated Learning
 
-This repo contains a proof of concept implementation of the
-coordinator in Rust.
+This repository contains the source code of the XAIN Federated
+Learning platform.
 
+XAIN provides privacy-preserving technology dedicated to keep the data
+used for the training of AI projects private. Our privacy engine for
+machine learning is compliant with data privacy regulations such as
+GDPR and CCPA. It offers a simple and scalable multi-party computation
+based on federated learning that reflects technical and regulatory
+needs of commercial AI projects.
 
-## Architecture
+## Resources
 
-This aggregator handles very different types of requests than the
-coordinator itself: weight distribution involves streaming high
-volumes of data over the network, and aggregation involves periodic
-CPU intensive tasks.
+- [Whitepaper](https://www.xain.io/federated-learning-technology)
+- [Source code](https://github.com/xainag/xain-fl/)
+- [Rest API reference](https://xain-fl.readthedocs.io/en/latest/api)
+- [docker images](https://hub.docker.com/repository/docker/xain/xain-fl/)
```

Comment:
When I open this link, the browser redirects me to the login page.
Can you try this link `https://hub.docker.com/r/xain/xain-fl`.

Reply:
Done. Btw we should have a README for the dockerhub page. Right now it just takes the README from the repo's root.